### DISCUSSION TITLE
Construir um blog com javascript ?

### DISCUSSION THREAD

**Discussion by talitaazevedo** [frontendbr/forum#1595] [Perguntas e Respostas]
Bom dia Devs! 

Estou estudando node react e react native,  e estou iniciando um projeto de construir um blog pessoal como projeto.  

Gostaria de desenvolver utilizando JS  no back e no frontend,  então minha dúvida é  existe o wordpress mas eu não gostaria de usa-lo,  eu vi na internet um pessoal que utiliza @Gatsby  para a desenvolver os blogs, vocês recomendam  ele ou alguma outra ferramenta/lib ?

**Comment by felipefialho** [comment]
Olá @tnazevedo, tudo bem?

Até hoje, todas as versões do meu site/blog foram construídas com JavaScript. 

Em 2013 usei:

- Docpad
- Less
- GruntJS

Em 2015:

- Harp / Jade
- Stylus
- Gulp

E agora, em 2019:

- Gatsby / React
- Styled Components
- Netlify

Todas as experiências foram muito boas, claro, é incomparável a agilidade que a stack atual me trouxe, a evolução com relação as anteriores é perceptível.

Sugiro ler esse texto que descrevi detalhadamente as coisas que falei ai em cima, [Como foi desenvolver meu novo blog usando o GatsbyJS]( https://www.felipefialho.com/blog/como-foi-desenvolver-meu-novo-blog-usando-o-gatsbyjs/) e também o [Making of](https://willianjusten.com.br/making-of-blog-novo/) do blog do @willianjusten, que me inspirou a refatorar o meu site.

Qualquer dúvida, avisa!

**Comment by Juuhz** [comment]
#1425

**Comment by talitaazevedo** [comment]
@Juuhz  Obrigada  eu lí o post relacionado inteiro foi muito esclarecedor.
@felipefialho  obrigada vou pesquisar mais a respeito, vou começar com desenvolvimento,  e  divulgar uns post sobre o andamento do projeto.
